modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -68,15 +68,18 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
 
   const getTableFromCell = (cell: SugarElement<HTMLTableCellElement>) => TableLookup.table(cell, isRoot);
 
+  const postExecute = (editor: Editor, table: SugarElement<HTMLTableElement>, rng: Range): void => {
```

Comment:
I don't think `editor` needs to be passed through as it is already in scope? Could possibly do something like this:
```suggestion
  const postExecute = (table: SugarElement<HTMLTableElement>) => (rng: Range): void => {
```

Then below it could be something like this:
```
execute(table, targets).each(postExecute(table));
```

Reply:
Yeah that's a nice change, thanks